src/rulesets/asyncapi/__tests__/asyncapi-operation-description.ts

```diff
@@ -1,31 +1,30 @@
-import { cloneDeep } from 'lodash';
-
-import { buildTestSpectralWithAsyncApiRule } from '../../../../setupTests';
-import { Rule } from '../../../rule';
 import { Spectral } from '../../../spectral';
+import { loadRules } from './__helpers__/loadRules';
+import { DiagnosticSeverity } from '@stoplight/types/dist';
```

Comment:
why from `types/dist`?

Reply:
WebStorm does it by default at times :laughing: 
Will change

Reply:
obscure software